markup-assign.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
```

Comment:
You have 2 files with the same content. Is this intended?

Reply:
I still do. Of course, index.html is the proper one. 

This happened before I did the git push - I removed the file and even deleted the branch, but it didn't update so I must have missed something at one point. I'll take care of that right now. :blush: